src/DependencyInjection/Configuration.php

```diff
@@ -41,6 +41,17 @@ public function getConfigTreeBuilder()
 
         $this->addBundlesSection($root);
 
+        $root
+            ->children()
+                ->arrayNode('ivory_ckeditor')
+                    ->addDefaultsIfNotSet()
+                    ->canBeEnabled()
+                    ->children()
+                        ->scalarNode('config_name')->defaultValue('cmf_sonata_phpcr_admin_integration')->end()
```

Comment:
is this configuration defined somewhere? afaik it needs to be in some template so that javascript can see it when initializing ckeditor? or is that part still in, but we lost the configuration option?